### DISCUSSION TITLE
Q-Select : pressing TAB-key after clicking on an option, sometimes picks the wrong value

### DISCUSSION THREAD

**Discussion by Dylan190774** [quasarframework/quasar#12357] [General - Components / Directives / etc]
While using a Q-select with `use-input` set, I noticed that sometimes after pressing the `TAB`-key the wrong value is picked.

I'm guessing it has something to do with the hovering over the options-list, which is apparently still active somehow, after clicking on an option. Even with `transition-hide` set tot 'none' this happens.

When you hold the mouse-pointer still for a little while after picking the option, this behaviour doesn't happen.

Has anybody noticed this before?

**Comment by rstoenescu** [comment]
The value that is currently highlighted when pressing TAB because the selected one.

**Reply by Dylan190774** [reply]
No, the value changes after pressing TAB or ENTER to something that you hovered over, while the options-list was closing, which is totally unexpected behaviour, since the value that was clicked on is displayed, but changes to something you never expect, when you leave the field with TAB or ENTER.

**Reply by rstoenescu** [reply]
"The value that is currently highlighted" means exactly that. Doesn't matter the way you highlighted (through arrow keys or mouse). This is what TAB key does, as a means for quickly selecting. Why would you want to actually close the menu with TAB instead of ESC.

**Reply by Dylan190774** [reply]
No, you don't understand!
Please repeat the steps in my bug report. 
You click the select open, pick an option with the mouse. The options list closes and the picked value is displayed correctly. 
But, when you would have moved the mouse too early up or down (right after clicking, within a second), somehow internally a DIFFERENT value is selected (but still not shows). And after you leave the field with TAB or ENTER, this OTHER value is displayed, coming out of nowhere, while you even have never seen it highlighted, because the options list was already invisible after clicking on the original option.

**Reply by rstoenescu** [reply]
Hmm, that's a key difference there. Thanks for the follow-up.
Fix will be available in Quasar v2.5.4
